.github/renovate.json

```
@@ -11,6 +13,24 @@
   "rebaseWhen": "conflicted",
   "semanticCommits": "disabled",
   "packageRules": [
+    {
+      "matchManagers": ["npm"],
+      "commitMessageTopic": "npm package `{{depName}}`",
+      "additionalBranchPrefix": "js/"
+    },
+    {
+      "enabled": true,
+      "groupName": "Block Protocol",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": [
+        "^@blockprotocol/",
+        "^block-",
+        "blockprotocol",
+        "create-block-app",
+        "mock-block-dock"
+      ],
+      "addLabels": "area: blocks"
```

Isn't this label added by the labeler flow anyway? I'd rather keep it in one location.
```suggestion
      ]
```

---

Let me confirm. Stupidly didn't really think about the `labeler` as I implemented this, but I 100% agree it would be best to have a single source of truth.

---

The list is not as big as your new list but we can update that instead:
https://github.com/hashintel/hash/blob/12fecc40c71cf21350af50e198d58c8baadaadfc/.github/labeler.yml#L27-L28

---

@TimDiekmann I agree with your principle of removing all the `addLabels` from this PR, where we can achieve the same thing through better rules in `labeler.yml`. Would you mind taking the sister task at some point in the future? Captured as GEN-40 (extremely low prio... I was only picking this low-prio ticket up in the first place as I needed to clear my mind with something mundane for ~20 before getting back to writing).

---

Sure! I added a back-link to this issue so I'm able to find it again when picking it up.